### PR TITLE
making a responsive signIn page

### PR DESCRIPTION
**Pull request by YashPatel1605**
i have made a signIn page responsive. please review and merge

### CODE REVIEW

File: signIn.html [hunk 1]
```diff
@@ -1,105 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>SignIn Page</title>
-    <link rel="icon" type="image/x-icon" href="Symbol.jpeg">
+    <title>signUp Page</title>
```
Comment (divyanshu1593) [inline signIn.html]: title should be signIn Page

File: signIn.html [hunk 2]
```diff
@@ -1,105 +1,68 @@
...
+        <div class="container my-5">
+            <div class="row py-4 header"><h2 style="color:rgba(37, 173, 208, 1);">SCALE</h2><h2>TECH</h2></div>
+            <div class="row my-5">
+                <div class="col-md-5">
+                    <img src="Symbol.jpeg"  class="mx-auto d-none d-md-block"id="left" alt="scaletech logo">
+                </div>
+                <div class="col-md-7">
+                    <h2>Sign in</h2>
+                <div class="form-group">
+                    <label for="email">Email</label>
```
Comment (divyanshu1593) [inline signIn.html:48]: here it should be email or username. so we should be able to login with either of two.

**Review by divyanshu1593 — CHANGES_REQUESTED** [review]

**Review by divyanshu1593 — APPROVED** [review]
Done 👍

File: signIn.html (hunk 2 above)
Comment (JeelPatel231) [reply signIn.html:48]: backlog: implement handling email or username in single field

**Review by JeelPatel231 — APPROVED** [review]
LGTM!

**Pull request merged**